packages/node/src/executors/webpack/webpack.impl.ts

```diff
@@ -81,6 +82,12 @@ export async function* webpackExecutor(
   if (options.generatePackageJson) {
     generatePackageJson(context.projectName, projGraph, options);
   }
+
+  // Delete output path before bundling
+  if (options.deleteOutputPath) {
+    deleteOutputDir(context.root, options.outputPath);
+  }
+
```

Comment:
This needs to be **_above_** the generatePackageJson stuff, s.t. the generated package json isn't deleted

Reply:
I'll make that change and also rebase to resolve that merge conflict.